### PR TITLE
RET-3123 ITHC R1.2 Case Progression flux changes

### PR DESCRIPTION
**Pull request by petermoores-version1**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RET-3123

### Change description ###
RET-3123 flux changes required to deploy R1.2 Case Progression images to ITHC environment.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: apps/et/et-sya/ithc-image-policy.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+apiVersion: image.toolkit.fluxcd.io/v1beta1
+kind: ImagePolicy
```
Comment (hsjhita) [inline apps/et/et-sya/ithc-image-policy.yaml:2]: need to add the image policy into automation/kustomization.yaml as well

Comment (petermoores-version1) [reply apps/et/et-sya/ithc-image-policy.yaml:2]: Thanks trying to do this whilst in meetings and being pinged constantly. Will sort @hsjhita.

**Comment by hsjhita** [comment]
also in ithc, need to add the ithc yamls for each service into the patchesStrategicMerges (they are commented out atm so can be uncommented)

**Review by thomast1906 — DISMISSED** [review]
Kustomize required for actual ITHC env of ET

See here - https://github.com/hmcts/cnp-flux-config/blob/ret-3123-ithc-case-progression-flux/apps/et/ithc/base/kustomization.yaml#L11

**Review by jordankainos — APPROVED** [review]
lgtm

**Pull request merged**